UPGRADING.md

```diff
@@ -8,6 +8,149 @@ As always, [submit issues](https://github.com/FortAwesome/Font-Awesome/issues/ne
 
 ---
 
+## 5.12.x/5.13.x to 5.14.0
+
+In version 5.12.0 and 5.13.0 some of the icons were assigned unicode values
+that were outside the Private Unicode Area (PUA). This caused problems with
+some desktop software and caused the icons to show up as Chinese, Japanese, or
+Korean (CJK) ideographs.
+
+The unicode values have been re-assigned to values within the PUA range.
+
+If you were using any of the following icons with pseudo-elements you will need
+to change the CSS `content` value to the new unicode value.
+
+| Icon name              | Old  | New  |
+| ---------------------- | ---- | ---- |
+| bacteria               | f959 | e059 |
+| bacterium              | f95a | e05a |
+| box-tissue             | f95b | e05b |
+| caravan                | f8ff | f8ff |
```

Comment:
@robmadole so apparently caravan didn't change and was not supposed to change because f8ff is the last available code in the PUA 

I can confirm that it is not an issue, we can remove caravan from the table, it was my mistake to include it

Ref: https://unicode.org/Public/UNIDATA/UnicodeData.txt
```
E000;<Private Use, First>;Co;0;L;;;;;N;;;;;
F8FF;<Private Use, Last>;Co;0;L;;;;;N;;;;;
```